api/resource.rb

```diff
@@ -341,7 +345,7 @@ def create_uri
       if @create_url.nil?
         if @create_verb.nil? || @create_verb == :POST
           collection_uri
-        elsif @create_verb == :PUT
+        elsif @create_verb == :PUT || @create_verb == :PATCH
```

Comment:
imo we may as well make the `self_link_uri` the fallthrough instead of an error, we validate the input above,

Reply:
:+1: